docs/resources/themes/owncloud-theme.yml

```diff
@@ -5,20 +5,20 @@ font:
       normal: notoserif-regular.ttf
       bold: notoserif-bold.ttf
       italic: notoserif-italic.ttf
-      bold_italic: notoserif-bold_italic.ttf
+      bold_italic: notoserif-bolditalic.ttf
```

Comment:
Why are the fonts touched here?

Reply:
Because the configuration file uses incorrect font file names.